bin/ffscreencast

```diff
@@ -859,6 +894,11 @@ while [ $# -gt 0  ]; do
 		-e[a-z]*)
 			OUTPUT_EXT="$(echo "$1" | $SED 's/^..//g')"
 			;;
+		-o*)
+			shift
+			NAME="$1" 
+            run_timer &
```

Comment:
Nit, this line is indented with spaces rather than tabs. The file in general uses tabs.

Reply:
This I saw in git diff, I always use tabs. I don't know how this occurred.

Reply:
I've opened https://github.com/Dark-Kernel/ffscreencast/pull/1 which targets your branch - if merged, it will resolve this issue.

Reply:
Done.